### ISSUE TITLE
Is they are same?

### ISSUE THREAD

**Issue by bubnenkoff** [vladdeSV/scone#5]
Hi! I would like to ask is this http://code.dlang.org/packages/bearlibterminal package and scone for the same tasks?

**Comment by vladdeSV** [comment]
Both **Simple Console** (scone) and **BearLibTerminal** (BLT) set out to achieve the same goal, but the approach is different.

Personally, I do not like the general way BLT is designed, and such, I have designed scone to be as simple as possible for the developers. To choose which one to use depends entirely on your desired workflow.

BLT seems to have been in development since late 2013, while scone only has been since late 2015. I would assume BLT is much more stable.

**tl;dr**: to answer your question: yes.

**Comment by vladdeSV** [comment]
Also, for simple enquiries as this, please send me an email instead of opening an issue.
